docs/6-integrations.md

```diff
@@ -26,10 +26,58 @@
 - When sending other theme output through twigshim_render() (e.g., other Paragraph bundles, blocks, node templates), you must set up the Twig template variables manually and pass them in to twigshim.
 
 ## Drupal 8
+
+While KalaStatic is a standalone application, you can integrate it quite well with Drupal 8. To have Drupal 8 use templates and components from KalaStatic, consider the following...
+
 ### Components module
-- Twig namespaces
-- How do I set up the variable mapping - theme functions etc
-- JS version of Druapl specfic Twig filters are included 
+
+The [Components module](https://www.drupal.org/project/components) will allow Drupal and Twig to share the same `@kalastatic` namespace. Once the module is installed, add the following to your theme's `.info.yml` file:
+
+```
+component-libraries:
+  kalastatic:
+    paths:
+      - path/to/kalastatic/src
+```
+
+This will allow the theme to reference Twig templates like:
+```
+{% include "@kalastatic/components/molecules/button/button.twig" %}
+```
+
+### Template Inclusion
```

Comment:
This section is kind of a duplicate of what is specced out for the components section. Not sure that we need it here, possibly just need to link to the components docs.